src/gui/Src/Utils/CachedFontMetrics.h

```diff
@@ -13,6 +13,7 @@ class CachedFontMetrics : public QObject
         : QObject(parent),
           mFontMetrics(font)
     {
+        memset(mWidths, 0, sizeof(mWidths));

```

Comment:
ahh that was the problem. At first I had this but I tried some things and then removed it again :disappointed: